docker/build-git-openssl.sh

```diff
@@ -32,16 +32,28 @@ apt-get install -y libcurl4-openssl-dev
 apt-get source git
 (
 	cd git*
+	# Change the libcurl4-gnutls-dev dep to libcurl4-openssl-dev
 	sed -i debian/control \
 		-e 's/libcurl4-gnutls-dev/libcurl4-openssl-dev/' \
 		-e '/TEST\s*=\s*test/d' ./debian/rules
+	# Strip out all git-xyz packages from the control file to speed up builds
+	sed -i debian/control \
+		-e '/^Package: git-man/,$d' ./debian/control
```

Comment:
@ScottGarman this doesn't match the comment?

Reply:
Perhaps I could have explained it better - the sed basically removes everything in the control file starting with the Package definition of git-man through the end of the file. I can improve this comment further if you think that would clarify things.